jekyll/_cci2/dedicated-hosts-macos.adoc

```diff
@@ -0,0 +1,92 @@
+---
+version:
+- Cloud
+---
+= Dedicated Hosts for macOS
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn about CircleCI's dedicated host resource class offering for macOS users.
+:icons: font
+:toc: macro
+:toc-title:
+
+== Dedicated Hosts for macOS
+
+The dedicated host resource class is a new macOS option for those developing, building, testing, and signing iOS, iPadOS, macOS, WatchOS, and tvOS applications using the Xcode IDE. These dedicated resources provide an isolated environment for increased security.
+
+This resource class requires a 24-hour minimum lease and runs on Intel-based Mac hardware.
+
+The identifier for the dedicated host resource is `metal` and supports the following Xcode images:
+
+- Xcode 13.0.0
+- Xcode 12.5.1
+- Xcode 12.4.0
+- Xcode 12.3.0
+- Xcode 12.2.0
+
+== Pricing and specs
+
+Once a dedicated host has been allocated, you have exclusive access to it for a minimum of 24 hours. If the dedicated host is already in use when a job is kicked off, an additional dedicated host is reserved (with its own 24 hour lease window).
+
+Each account can currently have a maximum of three concurrent dedicated hosts. Any time over the initial 24 hours that a dedicated host is in use will be charged a per-minute overage rate (see table below for pricing details).
```

Comment:
The default maximum is now 1. We should add a note that they can increase that amount by reaching out to support or their account rep.

Reply:
We also don't need to worry about the overage rate, since we've simplified the pricing to be 100 credits/minute (with a 24-hour minimum).